_wallets/trezorone.md

```diff
@@ -2,20 +2,20 @@
 # This file is licensed under the MIT License (MIT) available on
 # http://opensource.org/licenses/MIT.
 
-id: trezor
-title: "Trezor"
-titleshort: "Trezor"
+id: trezorone
+title: "Trezor One"
+titleshort: "Trezor One"
 compat: "hardware"
 level: 2
 platform:
   - hardware:
     name: hardware
     os:
       - name: hardware
-        text: "wallettrezor"
+        text: "wallettrezorone"
         link: "https://trezor.io/"
         source: "https://github.com/trezor/trezor-firmware"
-        screenshot: "trezor.png"
+        screenshot: "trezorone.jpg"
```

Comment:
Please include the screenshot as a `png` instead of a `jpg`.

Reply:
0194369187cca5057e4a96fdcff57cab5b632a30